README.md

```diff
@@ -11,20 +11,19 @@ A Kubernetes operator for managing CloudFormation stacks via `kubectl` and a cus
 
 # Deploy to a cluster
 
-You need API access to a cluster running at least Kubernetes v1.7.
+You need API access to a cluster running at least Kubernetes v1.17.0+.
 
 Start the CloudFormation operator in your cluster by using the provided manifests:
 
 ```console
-$ kubectl create -f deploy/rbac.yaml
-$ kubectl create -f deploy/crd.yaml
-$ kubectl create -f deploy/operator.yaml
+$ make deploy IMG=quay.io/cuppett/cloudformation-operator
```

Comment:
We should align on the same image location in the docs.

Reply:
Yeah, that just slipped in there. I can clean that up. Previously, it referenced versions that maybe would only exist after it was build/pushed (e.g. tag 0.8). Okay if I use the previous location but the :latest tag or something?

Reply:
Done! Found a couple other spots too.

Reply:
Yes, that's fine. Thanks!